include/ibm/flight_recorder.hpp

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+
+#define FLIGHT_RECORDER_MAX_ENTRIES 300
+
+namespace bmcweb
+{
+namespace flightrecorder
+{
+using FlightRecorderData = nlohmann::json;
+using FlightRecorderTimeStamp = std::string;
+using FlightRecorderRecord =
+    std::tuple<FlightRecorderTimeStamp, FlightRecorderData>;
+using FlightRecorderCassette = std::vector<FlightRecorderRecord>;
+static constexpr auto flightRecorderDumpPath = "/tmp/redfish_events_flight_recorder";
+
+std::string getCurrentSystemTime()
+{
+    using namespace std::chrono;
```

Comment:
`#include <chrono>` 
Include what you use

Reply:
Upstream we would not allow this `using namespace std::chrono;`
I am fine with `using chrono std::chrono` if you want